src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataAnnotationsModelValidator.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
 {
-    public class DataAnnotationsModelValidator : IModelValidator, IClientModelValidator
+    public class DataAnnotationsModelValidator : IModelValidator
```

Comment:
where is this class now used?  if adapters aren't needed at all server-side, remove